src/linux/FAnotifyFsBackend.cc

```diff
@@ -0,0 +1,26 @@
+#include <memory>
+
+#include <poll.h>
+#include <unistd.h>
+#include <fcntl.h>
+#include <sys/stat.h>
+#include <sys/fanotify.h>
+
+#include "FAnotifyFsBackend.hh"
+
+void FAnotifyFsBackend::start() {
+  throw std::runtime_error("not implemented");
```

Comment:
I don't know if we'd want to have an unimplemented backend here. And I'm unsure if we'd even end up implementing this one due to the elevated system permissions that it requires (`CAP_SYS_ADMIN`)

Reply:
fixed
'fanotify-fs' backend is completely removed from the code